perses/dispersed/relative_setup.py

```diff
@@ -411,6 +411,8 @@ def __init__(self, topology_proposal, pos_old, new_positions, use_dispersion_cor
         #reverse functions to get a symmetric protocol
         self._reverse_functions = {param : param_formula.replace("lambda", "(1-lambda)") for param, param_formula in self._forward_functions.items()}
 
+        #setup splitting string:
+        self._splitting_string = "V R O H R V"
```

Comment:
There is a typo here: It should be `V R O H O R V`. You're missing the second `O`.

Reply:
That's not a typo, as noted [here](https://github.com/choderalab/openmmtools/issues/329).